pkg/controller/statefulset/stateful_set.go

```diff
@@ -388,12 +388,12 @@ func (ssc *StatefulSetController) processNextWorkItem() bool {
 	if quit {
 		return false
 	}
-	defer ssc.queue.Done(key)
```

Comment:
This is incorrect. Done must be called for the key here or Get will never return the key again

/close